posts/when_to_fold_em.md

```diff
@@ -0,0 +1,48 @@
+# When to fold 'em
+
+Some years ago, I worked as part of a development team on a project to upgrade a tangled legacy system managing a company’s payment systems. Two months in, and we had very little to show for it - pages of diagrams that looked like spiderwebs, a few outages caused by failed attempts to untangle the pile, and a rising level of frustration.
+
+That company had spent thousands of pounds, not to mention missed out on a bunch of other opportunities - and not got a good return on that investment. Stakeholders weren’t any happier than we were, and it was time to decide what to do next.
+
+> We’ve learned some things about what we’re trying to tackle here. Let’s give it another couple of months and see if we can make any progress.
+
+Of course, a month later, nothing had changed - it was the same team, with the same tools, and the same level of support.
+
+> We’ve already sunk so much money into this. We can’t throw that away now. Let’s keep fighting!
+
+This is the **sunk cost** fallacy in action - treating what you’ve already lost as a factor in the decision on what to do next.
+
+Our brains were compiled in caves, and some of its software is very out-of-date. Today, we’re going to do some bug-fixing.
```

Comment:
our brain* was compiled in caves and some of its software? our brains + their?

Reply:
Ah, good spot. Think the former is better.

Reply:
And then I immediately think the latter might be better. Hmm. Might just reword it a bit...

Reply:
> Our brains were compiled in caves, and some of **the** software is very out-of-date. Today, we’re going to do some bug-fixing.

Went through some other ways of writing it, but they're less pithy. I think this is simple edit is the best of a bad bunch.

Given the choice, I'd break the rule here for the sake of a better-sounding sentence - I don't think the clarity suffers even if's technically incorrect.